spec/acceptance/suites/default/redis_multi_instances_one_host_spec.rb

```diff
@@ -1,28 +1,44 @@
 require 'spec_helper_acceptance'
 
-describe 'redis::instance' do
+describe 'redis::instance example' do
+  instances = [6379, 6380, 6381, 6382]
   case fact('osfamily')
   when 'Debian'
     config_path = '/etc/redis'
     redis_name = 'redis-server'
   else
-    redis_name = 'redis'
     config_path = '/etc'
+    redis_name = 'redis'
   end
 
   it 'runs successfully' do
     pp = <<-EOS
-    class { 'redis':
-      default_install => false,
+    Exec {
+      path => [ '/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin', ]
     }
 
-    redis::instance {'redis1':
-      port => 7777,
+    $listening_ports = #{instances}
+
+    class { '::redis':
+      default_install => false,
+      service_enable  => false,
```

Comment:
If you disable this, could you add a test `describe service(redis_name) { it { is_expected.not_to be_enabled } }` and the same for the line below?

Reply:
Sure, will do.